README.md

```diff
@@ -2,3 +2,13 @@ Jitsi Meet
 ----------
 
 Flatpak wrapper around the official Jitsi Meet desktop application - https://github.com/jitsi/jitsi-meet-electron
+
+## Wayland support
+
+Jitsi Meet's Wayland support is currently experimental, and some features like screensharing are broken.
+
+Wayland support can be enabled by setting `JITSI_USE_WAYLAND=1` in [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal). It can also be enabled on the command line, like so:
```

Comment:
This line feels a bit odd, I'd suggest writing it something more like this, to be more clear about what's being done - and less like Flatseal is a required component.

```suggestion
Wayland support can be enabled by setting the environment variable `JITSI_USE_WAYLAND=1` either using [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal), or the command line, like so:
```